services/fluent-bit/0.23.0/defaults/cm.yaml

```diff
@@ -1,7 +1,7 @@
 apiVersion: v1
 kind: ConfigMap
 metadata:
-  name: fluent-bit-0.21.6-d2iq-defaults
+  name: fluent-bit-0.23.0-d2iq-defaults
```

Comment:
The image needs to be bumped few lines below at [#L11](https://github.com/mesosphere/kommander-applications/pull/957/files#diff-27c8bd355f8d30c860b7ac89922587fc47ea5870c8761e988294a47d7ecce15eL11)

Reply:
Please bump there^ as suggested and https://github.com/mesosphere/kommander-applications/blob/main/services/logging-operator/3.17.10/defaults/cm.yaml#L102 as well